css/css-pseudo/first-letter-1cchar-ref.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html  lang="en" >
+<head>
+<meta charset="utf-8"/>
+<title>CSS3 Pseudo: latin, letter + diacritic</title>
+<meta name="assert" id="assert" content=":first-letter will select a letter + a following combining character at the start of a line.">
+<link rel="help" href="https://drafts.csswg.org/css-pseudo-4/#first-letter-pseudo">
```

Comment:
@xfq I believe that ref files are supposed to have no metadata, but i'm pretty sure that they aren't supposed to have help metadata.

Reply:
Thanks for the reminder! I just removed them.

Reply:
I checked https://web-platform-tests.org/writing-tests/reftests.html#example-reftests and it says that all metadata should be removed. So i think we'll need to also remove the ```<link rel="author"``` metadata.

Reply:
Done.